docs/quickstart.md

```diff
@@ -0,0 +1,40 @@
+## Quickstart
+
+These instructions install the bare minimum that is required to run the MaxScale system test setup. This configuration requires about 10GB of memory to run.
+
+### Install MDBCI dependencies, MDBCI and configure MDBCI
+
+1. Install dependencies
```

Comment:
This is also present in requirements.md. Avoid such types of duplication in the documentation as you do in the code.